pkg/cmd/root/root.go

```diff
@@ -46,17 +46,25 @@ func NewRootCommand(f *factory.Factory, version string) *cobra.Command {
 	var help bool
 	fs.BoolVarP(&help, "help", "h", false, f.Localizer.MustLocalize("root.cmd.flag.help.description"))
 
+	var devpreview *string = nil
+	cmd.Flags().StringVar(devpreview, "devpreview", "", f.Localizer.MustLocalize("root.cmd.flag.devpreview.description"))
+	_, _ = profile.EnableDevPreview(f, devpreview)
+
 	// Child commands
 	cmd.AddCommand(login.NewLoginCmd(f))
 	cmd.AddCommand(logout.NewLogoutCommand(f))
 	cmd.AddCommand(kafka.NewKafkaCommand(f))
-	cmd.AddCommand(registry.NewServiceRegistryCommand(f))
 	cmd.AddCommand(serviceaccount.NewServiceAccountCommand(f))
 	cmd.AddCommand(cluster.NewClusterCommand(f))
 	cmd.AddCommand(status.NewStatusCommand(f))
 	cmd.AddCommand(completion.NewCompletionCommand(f))
 	cmd.AddCommand(whoami.NewWhoAmICmd(f))
 	cmd.AddCommand(cliversion.NewVersionCmd(f))
 
+	// Dev preview commands
+	if profile.DevPreviewEnabled(f) {
```

Comment:
I have investigated using custom help message template based on the annotations, however that will require pulling some large parts of the internal cobra functions etc. so decided to simplify it by listing those commands bellow. This will be mostly seen by us so there is no big deal with it.